### DISCUSSION TITLE
Sustained and Boost parameters

### DISCUSSION THREAD

**Discussion by eugenov** [Valkirie/HandheldCompanion#400] [Q&A] [answered]
Hello!
When I set sustained and boost limits in a profile, my app don't notice boost scale, it respect only sustained settings and keeps tdp around sustained level. Is boost tdp - for turboboost only? Not min and max tdp for my app?

**Comment by CasperH2O** [comment, marked as answer]
@eugenov I realize this is a bit late, but this should hopefully answer your question.

Recent HC versions have combined sustained and boost limits in the profiles. 

As for the range of possible TDP setting, in the main GUI Settings page at the bottom you will find cTDP and you can increase or decrease the range at your own risk.
